docs/topics/db/optimization.txt

```diff
@@ -411,4 +416,105 @@ objects to reduce the number of SQL queries. For example::
     my_band.members.add(me)
     my_band.members.add(my_friend)
 
-...where ``Bands`` and ``Artists`` have a many-to-many relationship.
+...where ``Band`` and ``Artist`` have a many-to-many relationship.
+
+If you have a list of objects, use the ``*``-operator when calling
+:meth:`~django.db.models.fields.related.RelatedManager.add`::
+
+    my_band.members.add(*new_members)
```

Comment:
I don't see much value in describing how to use ``*`` in Python (see [comment](https://github.com/django/django/pull/11597#discussion_r307693994)), please remove these paragraphs.